packages/web-api/src/types/request/slacklists.ts

```diff
@@ -0,0 +1,12 @@
+import type { Block, KnownBlock } from '@slack/types';
+
+import type { TokenOverridable } from './common';
+
+export interface SlackListsCreateArguments extends TokenOverridable {
+  name: string;
+  description_blocks?: Array<Record<string, Block | KnownBlock>>;
+  schema?: Array<Record<string, Block | KnownBlock>>;
```

Comment:
```suggestion
  schema?: Array<Record<string, unknown>>;
```

🚧 issue: I'm not sure this `schema` is using blocks as we know. I made a few comments on https://github.com/slackapi/python-slack-sdk/pull/1772 related to this, but perhaps we extract list schemas and list item schemas into new types?